### ISSUE TITLE
Does not install with Vite 3.0

### ISSUE THREAD

**Issue by valtism** [richardtallent/vite-plugin-singlefile#46]
Getting the following error on install:

```
❯ npm i vite-plugin-singlefile -D
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: test@0.0.0
npm ERR! Found: vite@3.0.2
npm ERR! node_modules/vite
npm ERR!   dev vite@"^3.0.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer vite@"^2.9.13" from vite-plugin-singlefile@0.10.0
npm ERR! node_modules/vite-plugin-singlefile
npm ERR!   dev vite-plugin-singlefile@"*" from the root project
```

Can be installed with --force, and seems to work fine, but would be nice if the peer dependencies were updated so this error doesn't occur.

**Comment by richardtallent** [comment]
Can't at the moment, but happy to take a PR. We should maintain 2.x as well since not everyone is moving to Vite 3 immediately.

**Comment by dcrobinett** [comment]
Force install suggestion worked great.  No issues at all so far.

**Comment by richardtallent** [comment]
Resolved in 0.11.0 :)
